sql/telemetry_derived/sync_send_tab_events_v1/view.sql

```diff
@@ -46,8 +43,14 @@ SELECT
     [device_id, event_category, event_method, event_object, server_time, flow_id],
     '-'
   ) AS insert_id,
-  time,
-  TIMESTAMP_MILLIS(time) AS timestamp,
```

Comment:
We remove `timestamp` as it is redundant and Amplitude folks indicate `time` in epoch millis is the canonical format.